### PR TITLE
Fix 178+ spelling errors

### PR DESCRIPTION
**Pull request by jsoref**
Hi. I have read https://github.com/thorvg/thorvg/blob/38852538f5474a130d3a1888b747fbc322d1baef/CONTRIBUTING.md. If the changes in this PR are of interest, then I'm happy to reformat the commits to match the guidelines.

This PR fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/thorvg/actions/runs/9706612820#summary-26790560612

The action will report that the changes in this PR would make it happy: https://github.com/jsoref/thorvg/actions/runs/9706613122#summary-26790561679

<details>
<summary>meson + ninja</summary>

```sh
jsoref@jsoref-mbp thorvg % meson . build -Dtests=true -Dloaders="all" -Dsavers="all" -Dbindings="capi" -Dtools="all" -Dlog=true; ninja -C build test
The Meson build system
Version: 1.4.1
Source dir: /Users/jsoref/code/thorvg/thorvg
Build dir: /Users/jsoref/code/thorvg/thorvg/build
Build type: native build
Project name: thorvg
Project version: 0.14.99
C++ compiler for the host machine: ccache c++ (clang 15.0.0 "Apple clang version 15.0.0 (clang-1500.3.9.4)")
C++ linker for the host machine: c++ ld64 1053.12
Host machine cpu family: aarch64
Host machine cpu: aarch64
Configuring config.h using configuration
Found pkg-config: YES (/opt/homebrew/bin/pkg-config) 0.29.2
Run-time dependency libpng found: YES 1.6.43
Run-time dependency libturbojpeg found: YES 3.0.3
Run-time dependency libwebp found: YES 1.4.0
Library pthread found: YES
Message: 

Summary:
    ThorVG version:             0.14.99
    Build Type:                 debugoptimized
    Prefix:                     /opt/homebrew
    Multi-Tasking:              true
    SIMD Instruction:           none
    Raster Engine (SW):         true
    Raster Engine (GL):         false
    Raster Engine (WG_BETA):    false
    Loader (TVG):               true
    Loader (SVG):               true
    Loader (TTF):               true
    Loader (LOTTIE):            true
    Loader (PNG):               true
    Loader (JPG):               true
    Loader (WEBP):              true
    Saver (TVG):                true
    Saver (GIF):                true
    Binding (CAPI):             true
    Binding (WASM_BETA):        false
    Log Message:                true
    Tests:                      true
    Examples:                   false
    Tool (Svg2Tvg):             true
    Tool (Svg2Png):             true
    Tool (Lottie2Gif):          true
    Extra (Lottie Expressions): true


Build targets in project: 6

thorvg 0.14.99

  User defined options
    bindings: capi
    loaders : all
    log     : true
    savers  : all
    tests   : true
    tools   : all

Found ninja-1.12.1 at /opt/homebrew/bin/ninja
WARNING: Running the setup command as `meson [options]` instead of `meson setup [options]` is ambiguous and deprecated.
ninja: Entering directory `build'
[288/289] Running all tests.
1/2 Capi Unit Tests        OK              0.36s
2/2 Unit Tests             OK              0.93s

Ok:                 2   
Expected Fail:      0   
Fail:               0   
Unexpected Pass:    0   
Skipped:            0   
Timeout:            0   

Full log written to /Users/jsoref/code/thorvg/thorvg/build/meson-logs/testlog.txt
```

</details>

### CODE REVIEW

**Comment by jsoref** [comment]
Fwiw, I've created a PR w/ these changes into a fork's `main` branch and everything passes, although there are bugs in some of the CIs themselves.

File: README.md
Comment (hermet) [inline README.md]: @jsoref correct?

File: src/renderer/sw_engine/tvgSwRle.cpp
Comment (hermet) [inline src/renderer/sw_engine/tvgSwRle.cpp:325]: This patch is incorrect.

`acount` -> `account`

**Review by hermet — APPROVED** [review]
Thanks for your contribution. Overall, it looks good to me. It would be nice if you could address some feedback.

**Comment by hermet** [comment]
@RuiwenTang Hello, could you please confirm `gl_engine` changes?
@SergeyLebedkin Hello, could you please confirm `wg_engine` changes?

**Comment by hermet** [comment]
The patch must be squashed before being pushed.

File: src/renderer/gl_engine/tvgGlRenderTask.h
Comment (RuiwenTang) [inline src/renderer/gl_engine/tvgGlRenderTask.h:153]: `getColorTexture` is more meaningful

File: src/renderer/sw_engine/tvgSwRle.cpp
Comment (jsoref) [reply src/renderer/sw_engine/tvgSwRle.cpp:325]: I think it's really `count` or `offset` and not an _account_. My default correction would have been to `account` because that's usually right...

File: README.md
Comment (jsoref) [reply README.md]: Sorry. Overzealous correction. Dropped.

**Comment by jsoref** [comment]
I'm happy to squash once people are happy with the contents (but then I'd need to work on a commit message).

Offhand, from https://github.com/thorvg/thorvg/blob/38852538f5474a130d3a1888b747fbc322d1baef/CONTRIBUTING.md#commit-message, it looks like it should be something like:

```
Fix typos
```

Certainly, that would more or less appear to follow the pattern of https://github.com/thorvg/thorvg/commit/915d2b9f935e9c9523c087194f7cd376e8eb193c.patch

File: src/renderer/sw_engine/tvgSwRle.cpp
Comment (hermet) [reply src/renderer/sw_engine/tvgSwRle.cpp:325]: agree, thanks.

**Comment by hermet** [comment]
> I'm happy to squash once people are happy with the contents (but then I'd need to work on a commit message).
> 
> Offhand, from https://github.com/thorvg/thorvg/blob/38852538f5474a130d3a1888b747fbc322d1baef/CONTRIBUTING.md#commit-message, it looks like it should be something like:
> 
> ```
> Fix typos
> ```
> 
> Certainly, that would more or less appear to follow the pattern of https://github.com/thorvg/thorvg/commit/915d2b9f935e9c9523c087194f7cd376e8eb193c.patch

You can just leave it to maintainers. btw, could you please update yourself here? https://github.com/thorvg/thorvg/blob/main/AUTHORS

**Review by SergeyLebedkin — APPROVED** [review]

**Review by RuiwenTang — APPROVED** [review]

**Pull request merged**
